src/screens/UScreenEditSub.pas

```diff
@@ -2152,7 +2178,7 @@ function TScreenEditSub.ParseInput(PressedKey: cardinal; CharCode: UCS4Char; Pre
         end;
 
       end; // case
-    end;
```

Comment:
Is this indentation change deliberate? (I only have the Github UI right now)

Reply:
fixed this as well